### DISCUSSION TITLE
Displaced sidebar elements on mobile platforms

### DISCUSSION THREAD

**Discussion by domsalisbury** [mmistakes/minimal-mistakes#2822] [Q&A]
On mobile platforms, the sidebar is of course no longer a sidebar, but rather appears horizontally under the masthead. I have an issue in that the elements are displaced to the right when no avatar is used (see attached photo, which shows my homepage).

Is there a quick way to get the name and bio aligned to the left of the page? I don't want any changes to the layout and positioning on desktop platforms.

TIA.

![Screenshot_20210210-212854](https://user-images.githubusercontent.com/58000752/107620982-79ff7b00-6c4d-11eb-8055-26ca6a3f5a49.png)

**Comment by domsalisbury** [comment]
Bumping this, as I never got an answer, and it's been 18 months!

**Reply by iBug** [reply]
Any chance you can share a repository and a live website so it's easier to take a look at?

**Reply by domsalisbury** [reply]
Sure. Live website is here: https://domsalisbury.github.io/ (You'll have to view this on a mobile device to see the issue).

Repository is here: https://github.com/domsalisbury/domsalisbury.github.io

It just looks really ugly with the follow button instead of the sidebar on mobile devices. It would look better if it wasn't displaced (note no avatar — which might be why it is displaced), but getting rid of the author profile stuff on mobile might be a solution.

**Reply by iBug** [reply]
Looks like you'll be able to get away with this by setting `padding-left` to zero for `.author__content`:

![image](https://user-images.githubusercontent.com/7273074/194139894-487d485e-d197-4d2c-b13c-797b75de3311.png)

Refer to [customizing CSS](https://mmistakes.github.io/minimal-mistakes/docs/stylesheets/#customizing) in the documentation for guidance.

**Reply by domsalisbury** [reply]
Now I'm really confused. I played about with this, and now it seems to be aligned left. I'm confused because in my current code I have not removed the 15px padding....

Are you able to check the site again @iBug? I'm not very good with html and css, and don't have a mobile emulator or whatever you call them!

**Reply by iBug** [reply]
Looking good.

Plus you don't even need a "mobile emulator". Just press F12 on your PC browser and hit the "mobile screen" button. Alternatively you can resize the viewport to a narrower area, as "responsive view" works on the viewport width and cares nothing about the actual device.

![image](https://user-images.githubusercontent.com/7273074/194268682-5303233a-1311-4db6-8ad7-772cf90f8d89.png)

**Reply by domsalisbury** [reply]
Thanks for your help. I'm on macos, but I will read up on how to test this.
